src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -20,14 +20,19 @@
 public class StorageManager extends ComponentManager implements Storage {
 
     private static final Logger logger = LogsCenter.getLogger(StorageManager.class);
+    private static final String BACKUP_ROUTE_TAIL = "-backup.xml";
+
     private AddressBookStorage addressBookStorage;
     private UserPrefsStorage userPrefsStorage;
+    private AddressBookStorage backupAddressbook;
 
 
     public StorageManager(AddressBookStorage addressBookStorage, UserPrefsStorage userPrefsStorage) {
         super();
         this.addressBookStorage = addressBookStorage;
         this.userPrefsStorage = userPrefsStorage;
+        this.backupAddressbook = new XmlAddressBookStorage(addressBookStorage.getAddressBookFilePath()+BACKUP_ROUTE_TAIL);
```

Comment:
white space after + is needed otherwise you will break Travis.
Can try ./gradle checkstyleMain to see if other checkStyle errors are still present